default.nix

```diff
@@ -9,16 +9,18 @@ let
 in
 rec {
 
-  hydra-cli = ((pkgs.callPackage ./Cargo.nix {
-    cratesIO = pkgs.callPackage ./crates-io.nix {};
-  }).hydra_cli {}).overrideDerivation(_: {
-    src = sources;
-    doCheck = true;
-    checkPhase = ''
-      echo "Checking formatting with 'rustfmt'"
-      find . -name "*.rs" | xargs ${rustfmt}/bin/rustfmt --check
-    '';
-  });
+  # XXX: we use buildPackageSingleStep instead of buildPackage, because the
```

Comment:
Is this true for any currently nix packaged cargo version or what is the matter there?